docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -88,7 +85,47 @@ gradient boosting methodologies.
 //end::regression-algorithms[]
 
 
-[discrete]
+[[dfa-regression-lossfunction]]
+===== Loss functions for {regression} analyses
+
+Loss function measures how well a given {ml} model fits the specific 
+data set. The loss function boils down all the different under- and 
+overestimations of the model to a single number. The bigger the difference 
+between the prediction and the ground truth, the higher the value of the loss 
+function. Loss functions are used automatically in the background during 
+<<hyperparameters,hyperparameter optimization>> and when training the decision 
+trees to compare the various iterations of the model and decide which iteration 
+performs better. 
```

Comment:
If you accept the suggestion for the first half of this sentence, then this bit is no longer required:
```suggestion

```